modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/05_enablement/page-definition.json

```diff
@@ -34,15 +148,281 @@
+												"marginRight": 0,
+												"marginTop": 0,
+												"paddingBottom": 0,
+												"paddingLeft": 0,
+												"paddingRight": 0,
+												"paddingTop": 0,
+												"widthType": "Fluid"
+											}
+										},
+										"pageElements": [
+											{
+												"definition": {
+													"widgetInstance": {
+														"widgetConfig": {
+															"articleId": "WEB-CONTENT-ENABLEMENT-02",
+															"assetEntryId": "47385",
+															"contentMetadataAssetAddonEntryKeys": "false",
+															"ddmTemplateKey": "",
+															"enableViewCountIncrement": "false",
+															"groupId": "",
+															"userToolAssetAddonEntryKeys": "false"
+														},
+														"widgetName": "com_liferay_journal_content_web_portlet_JournalContentPortlet",
+														"widgetPermissions": [
+															{
+																"actionKeys": [
+																	"VIEW"
+																],
+																"roleKey": "Guest"
+															},
+															{
+																"actionKeys": [
+																	"VIEW"
+																],
+																"roleKey": "Site Member"
+															},
+															{
+																"actionKeys": [
+																	"PERMISSIONS",
+																	"PREFERENCES",
+																	"CONFIGURATION",
+																	"VIEW",
+																	"ADD_TO_PAGE"
+																],
+																"roleKey": "Owner"
+															}
+														]
+													}
+												},
+												"type": "Widget"
+											}
+										],
+										"type": "Section"
+									}
+								],
+								"type": "Column"
+							},
+							{
+								"definition": {
+									"columnViewports": [
+										{
+											"columnViewportDefinition": {
+												"size": 12
+											},
+											"id": "landscapeMobile"
+										},
+										{
+											"columnViewportDefinition": {
+											},
+											"id": "portraitMobile"
+										},
+										{
+											"columnViewportDefinition": {
+											},
+											"id": "tablet"
+										}
+									],
+									"size": 4
+								},
+								"pageElements": [
+									{
+										"definition": {
+											"fragmentStyle": {
+												"borderWidth": "0",
+												"textAlign": "",
+												"width": "388px"
+											},
+											"indexed": true,
+											"layout": {
+												"borderWidth": 0,
+												"marginBottom": 0,
+												"marginLeft": 0,
+												"marginRight": 0,
+												"marginTop": 0,
+												"paddingBottom": 0,
+												"paddingLeft": 0,
+												"paddingRight": 0,
+												"paddingTop": 0,
+												"widthType": "Fluid"
+											}
+										},
+										"pageElements": [
+											{
+												"definition": {
+													"widgetInstance": {
+														"widgetConfig": {
+															"articleId": "WEB-CONTENT-ENABLEMENT-03",
+															"assetEntryId": "47394",
```

Comment:
This field would be replaced by site initializer?

Reply:
Yes, the site initializer will locate the journal article because it is using the articleId as reference.